src/pkg/logger/logger.go

```diff
@@ -131,6 +134,13 @@ func addFlags(fs *pflag.FlagSet, defaultFile string) {
 		MaskSensitiveDataFN,
 		false,
 		"anonymize personal data in log output")
+
+	fs.BoolVar(
+		&logStorageEngineFV,
+		logStorageEngineFN,
+		false,
+		"allow kopia to log to the corso log too. Uses the same log level as the corso logger")
```

Comment:
Nit: to keep it generic rather than kopia specific.
```suggestion
		"include logs produced by the downstream storage systems")
```